models/staging/stg_dbt_tests.sql

```diff
@@ -22,9 +22,19 @@ flatten as (
         node.value:schema::string as model_schema,
         node.value:name::string as test_name,
         node.value:package_name::string as package_name,
-        node.value:path::string as model_path,
+        node.value:path::string as model_path_prep,
         node.value:checksum.checksum::string as checksum,
-        node.value:config.materialized::string as model_materialization
+        node.value:config.materialized::string as model_materialization,
+
+         case
+            when model_path_prep ilike '%bespoke_tests/%'
+                then replace(model_path_prep, 'bespoke_tests/', '')
+            when model_path_prep ilike '%schema_test/%'
+                then replace(model_path_prep, 'schema_test/', '')
+            when model_path_prep ilike 'data_test/%'
+                then replace(model_path_prep, 'data_test/', '')
+            else model_path_prep
+        end as model_path
```

Comment:
this is a liiiiitle redundant since theres a replace for bespoke tests in `fct_dbt_test_executions` (but you can probably remove that one and keep this one (potentially as embedded replace statements? i actually dont know if one way. is better than the other or not))

![image](https://user-images.githubusercontent.com/15128477/150461602-3ef88902-1bde-4ec0-a31e-d3c2b1264931.png)

Reply:
yes doing this! for some reason the case when statement was only working properly when it was in both models?? but the embedded replace() works perfectly